custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
+  title: "RAIA",
+  text: "Software Developer • Not a Robot",
 };
 
 const description = {
-  text: faker.lorem.paragraph(),
```

Comment:
Consider using template literals for string interpolation instead of concatenation for improved readability.

Suggested change:
```javascript
`Bringing exploratory mindset to technical leadership, with six years of experience in software development and computational research`
```

For reference, see [MDN Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).